### PR TITLE
Feature/98 contact method

### LINKED ISSUE #98: Method of contact does not show on frontend

### PR DESCRIPTION
**Pull request by jshor**
This feature fixes #98 - allows the user to specify the contact method if they select "Other" as the mode of contact in the comment form

### CODE REVIEW

File: server/models/CommentModel.js [hunk 1]
```diff
@@ -22,6 +22,10 @@ module.exports = db.define('comment', {
 
   body: {
     type: Sql.TEXT
-  }
+  },
+
+  // contactMethod: {
```
Comment (jackofseattle) [inline server/models/CommentModel.js]: I don't think you meant to comment this out. this will cause issues with the db in production

**Comment by jackofseattle** [comment]
LGTM except for the one odd comment

**Comment by jackofseattle** [comment]
you now have merge conflicts, please address them and the comments here and merge.

File: server/models/_connection.js [hunk 2]
```diff
@@ -15,6 +15,6 @@ if (dbUrl) {
   module.exports = new Sequelize('sqlite://' + dbPath, undefined, undefined, {
     dialect: 'sqlite',
     storage: dbPath,
-    logging: process.env.DISABLE_SQL_LOGGING ? false : undefined
+    logging: process.env.DISABLE_SQL_LOGGING === 'true' ? true : false
```
Comment (jackofseattle) [inline server/models/_connection.js]: This logic is inverted.  if disable is true then you are setting logging to true..

Comment (jshor) [inline server/models/_connection.js]: ...I thought I fixed this...?

**Pull request merged**